sktime/forecasting/hf_transformers_forecaster.py

```diff
@@ -227,6 +231,31 @@ def _fit(self, y, X, fh):
         elif self.fit_strategy == "full":
             for param in self.model.parameters():
                 param.requires_grad = True
+        elif self.fit_strategy == "lora":
+            peft_config = LoraConfig(
+                r=8,
```

Comment:
The user would probably like to have control about the peft_configs. I.e., by providing a dict of parameters that is passed to the LoraConfig. 

```python
LoraConfig(**self.peft_config_dict)
```

Reply:
sure makes sense, 2 things:
1. no need to give any default value to param `peft_config_dict`, right?
2. should I also create an example with peft in docstring?